src/translations/default/about.mdx

```diff
@@ -0,0 +1,45 @@
+**CONTENT_NOT_TRANSLATED**
```

Comment:
Is this where the English translation is supposed to happen? Should this stay this way?

Reply:
As discussed every english translations are located in `translations/en/` folder

Reply:
This default folder is due to a know problem on `next-i18next`. This is really bad and should be removed

The only way I found was to switch to next-translate which is also a lot easier to use.

We can try that in a next iteration